BedrockServer.h

```diff
@@ -254,4 +243,23 @@ class BedrockServer : public SQLiteServer {
     // Pointer to the control port, so we know which port not to shut down when we close the command ports.
     Port* _controlPort;
     Port* _commandPort;
+
+    // The following variables all exist to to handle commands that seem to have caused crashes. This lets us broadcast
+    // a command to all peer nodes with information about the crash-causing command, so they can refuse to process it if
+    // it gets sent again (i.e., if an end-user clicks 'refresh' after crashing the first node). Because these can
+    // originate in worker threads, much of this is synchronization code to make sure the sync thread can send this
+    // message before the worker exits.
+
+    // A shared mutex to control access to the list of crash-inducing commands.
+    shared_timed_mutex _crashCommandListMutex;
```

Comment:
Can you rename `_crashCommandListMutex` to be `_crashCommandsMutex`?  It's not actually a "list", and might as well just match the variable name it's controlling.